core-web/libs/edit-content/src/lib/fields/dot-edit-content-radio-field/dot-edit-content-radio-field.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="field">
+    <label
+        [attr.data-testId]="'label-' + field.variable"
+        [for]="field.variable"
+        [checkIsRequiredControl]="field.variable"
+        dotFieldRequired
+        >{{ field.name }}</label
+    >
```

Comment:
we don't need to repeat this in any component, that's what the field component is for, I did some changes to field component to avoid this check my PR

Reply:
All posibles fields can have label-hint?

Reply:
Yes.